### PR TITLE
ssh client copy_file bidirectional instead of always assuming local to remote copy

### PR DESCRIPTION
**Pull request by margaret**
It still defaults to assuming local->remote, so previous uses of it shouldn't be affected

### CODE REVIEW

File: dcos_test_utils/ssh_client.py [hunk 1]
```diff
@@ -54,21 +54,26 @@ def command(self, cmd: list, **kwargs) -> bytes:
...
+            if os.path.isdir(src):
+                copy_command.append('-r')
+            remote_full_path = '{}:{}'.format(self.target, dst)
+            copy_command += [src, remote_full_path]
+        else:
+            remote_full_path = '{}:{}'.format(self.target, src)
+            copy_command += [remote_full_path, dst]
         cmd = ['scp'] + self.opt_list + ['-P', str(self.port)] + copy_command
-        log.debug('Copying {} to {}'.format(src, remote_full_path))
+        log.debug('Copying {} to {}'.format(*copy_command))
```
Comment (colin-msphere) [inline dcos_test_utils/ssh_client.py]: Does this log statement work if `copy_command` includes the `-r`? That would be more than 2 arguments?

Comment (margaret) [reply dcos_test_utils/ssh_client.py]: Ahh good catch, that wouldn't work. It doesn't throw an exception but it would log `-r to src`

**Comment by margaret** [comment]
Not sure why the tests aren't reporting. I triggered them manually.

✅ **Tox** https://teamcity.mesosphere.io/viewLog.html?buildId=1021388&tab=buildResultsDiv&buildTypeId=DcosIo_DcosTestUtils_ToxIntegrationTestEnterprise
🔄  **Integration test Open** https://teamcity.mesosphere.io/viewLog.html?buildId=1021389&tab=buildResultsDiv&buildTypeId=DcosIo_DcosTestUtils_ToxIntegrationTest
✅ **Integration test Enterprise** https://teamcity.mesosphere.io/viewLog.html?buildId=1021388&tab=buildResultsDiv&buildTypeId=DcosIo_DcosTestUtils_ToxIntegrationTestEnterprise

**Comment by margaret** [comment]
🤔 ❓ and the CI suddenly ran again and is showing up

**Comment by margaret** [comment]
Oooo that's what the 'update branch' button does. Updates with master.

**Pull request merged**